Resources/views/Collector/db.html.twig

```diff
@@ -40,9 +66,15 @@
                 <b>Cache puts</b>
                 <span class="sf-toolbar-status sf-toolbar-status-{{ collector.cachePutsCount > 0 ? 'yellow' : 'green' }}">{{ collector.cachePutsCount }}</span>
             </div>
+        {% else %}
+            <div class="sf-toolbar-info-piece">
+                <b>Second Level Cache</b>
+                <span class="sf-toolbar-status sf-toolbar-status-red">disabled</span>
```

Comment:
we should avoid displaying a red box for people using Doctrine ORM 2.4 as they cannot enable the SCL if it does not exist.